ci-operator/config/openshift/tektoncd-triggers/openshift-tektoncd-triggers-release-v0.3.1.yaml

```diff
@@ -0,0 +1,54 @@
+base_images:
+  base:
+    cluster: https://api.ci.openshift.org
+    name: "4.3"
```

Comment:
I think it would be good to have (in a next PR) a release job that tests on 4.4 ? like we do for other components?